TrackParamFilter/sections/specific.txt

```diff
@@ -220,3 +220,7 @@
 ||websearch.rakuten.co.jp/Web?$removeparam=ref
 ||rakuten.co.jp^*&trflg=$removeparam=trflg
 ||search.yahoo.co.jp^$removeparam=fr
+! Daraz & Shop (Samesite)
```

Comment:
`||*.` is redundant. Better `||daraz.$......`

Reply:
Fixed.